### DISCUSSION TITLE
Small white space at bottom of bootstrap site

### DISCUSSION THREAD

**Discussion by dn3hgh** [twbs/bootstrap#39639] [Q&A] [answered]
I have tried over and over debugging with chrome tools. When I hover over the "P" element in my footer it shows that it is pushing down and creating the space. However, I have no Margin bottom (mb- is the bootstrap for margin bottom) in my footer at all. 

**Here is the element that Chrome Tools says is the problem.**
`<p class="text-white pb-5 fs-3">Copyright © Frontend Bootcamp</p>`

**And here is the Netlify link to it so you can use Chrome tools.**
(https://melodious-sundae-96dca8.netlify.app/)

thanks

**Comment by julien-deramond** [comment, marked as answer]
Hey @dn3hgh 
By default, `<p>` has a margin-bottom of `1rem`. Add to this `<p>` a `.mb-0` (`margin-bottom: 0`) and the white area will disappear.

**Reply by dn3hgh** [reply]
Thanks so much I put in mb-0 and it works.
